public/javaScript/MessageHandler.js

```diff
@@ -1,8 +1,25 @@
 const HOST = location.origin.replace(/^http/, 'ws');
 const ws = new WebSocket(HOST);
+
 const content = $('#content');
+const status = $('#status');
+const input = $('#input');
```

Comment:
could we maybe refactor the code and go with 
```
const Host = location.origin.replace(/^http/, 'ws'),
          ws = new WebSocket(HOST),
         content = $('#content');

```

Reply:
Good recommendation, will refactor and push an updated version

Reply:
good stuff.👍